README.md

```diff
@@ -69,12 +69,11 @@ npm test
 Start the server in development mode
 
 ```sh
-OAUTH_AUTHORIZATION_URL="https://login.$DEPLOY_ENV.dev.cloudpipeline.digital/oauth/authorize" \
-OAUTH_TOKEN_URL="https://uaa.$DEPLOY_ENV.dev.cloudpipeline.digital/oauth/token" \
```

Comment:
It seems like this commit could be squashed with `Create clients when needed` which removed the use of these variables.

There's also a reference to these environment variable names in `src/main.test.ts`.